### PR TITLE
[#740] - Adding k8s endpoint to airy controller

### PR DESCRIPTION
**Pull request by paulodiniz**

### CODE REVIEW

**Review by chrismatix — DISMISSED** [review]
Very cool!

File: infrastructure/helm-chart/charts/controller/templates/service.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  namespace: {{ .Values.global.namespace }}
+spec:
+  ports:
+    - name: web
+      port: 80
+      targetPort: 8080
+      protocol: TCP
+  type: NodePort
+  selector:
+    app: controller-info
```
Comment (ljupcovangelski) [inline infrastructure/helm-chart/charts/controller/templates/service.yaml:14]: should be `app: airy-controller`

File: infrastructure/helm-chart/charts/controller/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: controller-info
```
Comment (ljupcovangelski) [inline infrastructure/helm-chart/charts/controller/templates/service.yaml:4]: I think that `airy-controller` would be a more appropriate name for the service.

**Review by ljupcovangelski — COMMENTED** [review]
I can confirm that it works
```
bash-4.4# curl controller-info.default/components
{"components":["api-admin","api-auth","api-communication","api-websocket","media-resolver","source-google","frontend-chat-plugin","frontend-ui","source-chat-plugin","source-facebook","source-twilio","integration-webhook"]}
```
:tada:

**Review by ljupcovangelski — APPROVED** [review]
Looks good :+1:

**Pull request merged**